src/components/PreviewFrame.jsx

```diff
@@ -9,6 +9,7 @@ import {sourceDelimiter} from '../util/generatePreview';
 const sandboxOptions = [
   'allow-forms',
   'allow-popups',
+  'allow-same-origin',
```

Comment:
(Incidentally, the inability to do a sandboxed client-side-only iframe is why Popcode doesn’t support IE/Edge at all.)